docs/source/user_guide/operators.rst

```diff
@@ -307,12 +307,8 @@ DPF provides three main types of operators:
 Operators for importing or reading data
 ***************************************
 
-These operators provide for reading data from solver files or from standard file types:
-
-- For MAPDL, supported solver file formats include RST, MODE, RFRQ, and RDSP.
-- For LS DYNA, D3PLOT files are supported.
-- For Fluent and CFX, CAS.H5, DAT.H5, RES, and FLPRJ files are supported.
-- For Abaqus, ODB files are supported.
+These operators provide for reading data from solver files or from standard file types
+such as .RST (MAPDL), .D3Plot (LS DYNA), .CFX (Fluent) or .OBD (Abaqus).
```

Comment:
This is a contradiction in terms. I would put: `.CAS.H5/.DAT.H5 (Fluent)`, `.CAS.CFF/.DAT.CFF (CFX)`